app/src/main/AndroidManifest.xml

```diff
@@ -7,7 +7,7 @@
     <application
         android:name=".MifosSelfServiceApp"
         android:allowBackup="true"
-        android:icon="@mipmap/ic_launcher"
+        android:icon="@drawable/ic_icon"
```

Comment:
![img_20170306_150712](https://cloud.githubusercontent.com/assets/16569570/23604534/43a570a0-027f-11e7-934e-61aa6348f88d.jpg)

Preview of how it looks in Android 6.0